CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contribution Guide
+
+Firstly, thank you for considering making a contribution to our project! It's people like you that make Operation Code such a great community.
+
+We're an open source project and we love receiving contributions from our community - you! There are many ways to contribute to our projects, from writing tutorials or blog posts, improving documentation, submitting bug reports or feature requests, or writing code which can be merged into any of our repositories.
+
+The team at Operation Code urges all contributors to join our Slack team. Participating in discussions with the community on our Slack channel, is the best way to run new ideas by the team, and is the best place to get help. You can get an invitation to our Slack channel by [requestion to join Operation Code](https://operationcode.org/join). Once in our Slack team, simply type: '/open #oc-projects' and then click enter. Feel free to ask for help; everyone is a beginner at first :smile_cat:!
+
+**This guide assumes that you have some at least some familiarity with submitting a pull request on Github. If you don't, that's ok too though. Simply start by reading Github's own user documentation on how to fork a repository, and make your own edits. That documentation is [here](https://help.github.com/articles/about-pull-requests/) and [here](https://help.github.com/articles/creating-a-pull-request/). 3rd party tutorials on Git/Github can be found [here](https://medium.freecodecamp.org/what-is-git-and-how-to-use-it-c341b049ae61) and [here](https://medium.freecodecamp.org/how-to-use-git-efficiently-54320a236369?source=linkShare-e41cd5edcdac-1535829065)**
+
+
+## Development Workflow
+
+### Dependencies
+
+The primary technologies used in this project are:
+
+- [Python](https://www.python.org)
+- [Flask](http://flask.pocoo.org)
+- [Postgres](https://www.postgresql.org)
+
+To work on the codebase for this project, you will need to have those dependencies installed.
+
+
+### Finding An Issue
+
+<details>
+	<summary>Click to Expand</summary>
+<ul>
+<li> After installing the listed dependencies, you can get to work coding on this project. A listing of this repo's can be found <a href="https://github.com/OperationCode/resources_api/issues">here</a></li>
+<li> Go to the issues page for this repo, [here]() and browse for an issue that you would like to work on. Don't be afraid to ask for help or clarification.</li>
```

Comment:
I think these two items basically say the same thing. Also, there's a broken link in the second list item. I think we should combine them. Perhaps something like:
```html
<li> After installing the listed dependencies, you can get to work coding on this project. A listing of this repo's issues can be found <a href="https://github.com/OperationCode/resources_api/issues">here</a>. Browse for an issue that you would like to work on. Don't be afraid to ask for help or clarification.</li>
```